tests/system_test.py

```diff
@@ -421,29 +437,71 @@ def attributes(e, level):
             # top level list of tuples ('section-name', dict)
             return "".join(["%s {\n%s}\n"%(n, attributes(p, 1)) for n, p in self])
 
-    def __init__(self, name=None, config=Config(), pyinclude=None, wait=True,
+    def __init__(self, name=None, config=Config(), pyinclude=None, wait=True, wait_for_server_log=True,
                  perform_teardown=True, cl_args=None, expect=Process.RUNNING):
         """
         @param name: name used for for output files, default to id from config.
         @param config: router configuration
         @keyword wait: wait for router to be ready (call self.wait_ready())
+        @param wait_for_server_log Use server log check for "wait ready"
+        @param perform_teardown
+        @param cli_args
+        @param expect
```

Comment:
Is this good for anything? mentioning the parameters but not giving any description?